react/features/invite/components/dial-in-info-page/DialInInfoApp.web.js

```diff
@@ -5,20 +5,25 @@ import { I18nextProvider } from 'react-i18next';
 import { isMobileBrowser } from '../../../base/environment/utils';
 import { i18next } from '../../../base/i18n';
 import { parseURLParams } from '../../../base/util/parseURLParams';
+import { DIAL_IN_INFO_PAGE_PATH_NAME } from '../../constants';
 import { DialInSummary } from '../dial-in-summary';
 
 import NoRoomError from './NoRoomError';
 
 document.addEventListener('DOMContentLoaded', () => {
     const { room } = parseURLParams(window.location, true, 'search');
+    const { href } = window.location;
+    const ix = href.indexOf(DIAL_IN_INFO_PAGE_PATH_NAME);
+    const url = (ix > 0 ? href.substring(0, ix) : href) + room;
```

Comment:
Do we want to get the host and protocol part only with the whole substring logic or we have an use case where we take part of the path as well?

If we want only the host and protocol maybe it will be easier:
```
const { host, protocol } = location;

// and then it will be:
`${protocol}://${host}/`
```